Sources/TensorFlow/Operators/Basic.swift

```diff
@@ -1325,42 +1325,75 @@ extension Tensor.IndexPath {
 //===------------------------------------------------------------------------------------------===//
 
 extension Tensor {
-  /// Returns `true` if the given axis is in the range `[-rank, rank)`.
+  /// Returns `true` iff `axis` can refer to an axis of `self`.
   @usableFromInline
-  internal func isAxisInRange<T: BinaryInteger>(_ axis: T) -> Bool {
+  internal func isValid<T: BinaryInteger>(axis: T) -> Bool {
     let axis = Int(axis)
     return axis >= -rank && axis < rank
   }
 
-  /// Returns `true` if the given scalar tensor is in the range `[-rank, rank)`.
+  /// Returns `true` iff all scalars can refer to an axis of `self`.
```

Comment:
Hmm, that ain't right; it would have to return `false` unconditionally!  Maybe 
```suggestion
  /// Returns `true` iff all elements of axes denote an axis of `self`.
```
Again, you might consider a name for the argument separate from its argument label.

Reply:
I think it reads a bit better with the separate argument name vs label. Picking a name for this one is tricky. I've come up with `dims` but not fully sure I like it yet.